### ISSUE TITLE
Can't delete bookmarks

### ISSUE THREAD

**Issue by helpdeskdan** [afonsotrepa/PocketGopher#8]
The "remove" button under "Edit" does not do anything for me, anybody else have this issue?

**Comment by afonsotrepa** [comment]
The remove button will only have an effect when you're editing an already existing bookmark. If you're creating a new bookmark and hit remove it'll simply exit the bookmark creation process.

Was this your problem?

**Comment by helpdeskdan** [comment]
No, I am unable to delete bookmarks when I long press on them, and hit delete.  I am trying to delete the defective bookmark gopher://gopher.floodgap.com/7/groundhog/us/zipcode, but it will not go away!  Thank you for your time.
